packages/core/upload/admin/src/components/EditFolderDialog/EditFolderDialog.js

```diff
@@ -129,18 +127,18 @@ export const EditFolderDialog = ({ onClose, folder, parentFolderId }) => {
 
   return (
     <>
-      <ModalLayout onClose={() => onClose()} labelledBy="title">
-        <EditFolderModalHeader isEditing={isEditing} />
+      <ModalLayout onClose={() => !isEditFolderLoading && onClose()} labelledBy="title">
```

Comment:
Why shouldn't I be able to close the modal, if the request is still running?

Reply:
because the request won't be cancel when closing, you can try it - it will close the modal then a few moments later will show the success. I can leave it like it was before but I thought it was cleaner this way

Reply:
Tbh. I don't think it is a problem. I'd put it the other way around: if the request takes a while it is confusing for the user if nothing is happening, because there is no visual feedback, so they might actually click several times ...